src/components/tx/SponsoredBy/index.tsx

```diff
@@ -18,7 +18,7 @@ const SponsoredBy = ({ relays, tooltip }: { relays: RelayResponse; tooltip?: str
     <Box className={css.sponsoredBy}>
       <SvgIcon component={GasStationIcon} inheritViewBox className={css.icon} />
       <Stack direction="column">
-        <Stack direction="row" spacing={0.5} alignItems="center" mb={1}>
+        <Stack direction="row" spacing={0.5} alignItems="center" mb={0.2}>
```

Comment:
Nit: I'd rather put the pixel value, 0.2 always require a bit of mental effort to follow.

Reply:
I think `0.2` is better than `1.6px`. It is reactive to any base spacing changes and the latter is also not super "clear" either. As the `spacing` prop on this component is also relevant to the base, I'd sooner leave this as is for now.